basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -3,8 +3,47 @@
  */
 package basiclibrary;
 
+import java.util.Arrays;
+
 public class Library {
-    public boolean someLibraryMethod() {
-        return true;
+
+    //Rolling dice
+    public static int[] roll(int n) {
+        int[] valueArr = new int[n];
+
+        //Roll the dice
+        int random = (int)Math.ceil(Math.random());
+        for( int i = 0; i < n; i++){
+            valueArr[i] = random;
+        }
+        System.out.println(Arrays.toString(valueArr));
+        return valueArr;
+    }
+
+    //Contains duplicates
+    public static boolean containsDuplicates(int[] testArr){
+        for(int i = 0; i < testArr.length; i++){
+            for(int k = i+1; i < testArr.length; k++){
+                if(k != i && testArr[i] == testArr[k]) {
```

Comment:
if `k = i + 1` then you don't need to check if `k != i`, `k` will never be the same value as `i`